FAQ Section

dev129 opened the pull request:
This code is for addressing the issue of creation of FAQ section, it addresses the issue number #128

dev129 committed:
FAQ Section

This code contains the infomration for FAQ section

dev129 committed:
Merge branch 'develop' into feat-FAQ

notsoocool commented:
You don't fix the font size and also you still have used semibold and bold font style.

notsoocool commented on components/global/header.tsx:
```diff
@@ -108,7 +108,6 @@ export const Header = () => {
             </div>
           </div>
         </div>
-      </div>
```
fix this file

dev129 replied:
The header file is now fixed.

dev129 commented:
Okay can we like discuss it together, or do you have an idea in mind ? Like how it should look, so that its easier for me to understand

notsoocool commented on app/images/FAQ Image.png:
change the picture its blured
used the icon or images from this
https://www.streamlinehq.com/illustrations/new-york-monoline

dev129 replied:
You are a nice man , I will surely implement this and give a fresh design

dev129 replied:
I have updated the image and removed the previous one , and that image is responsive to dark and light mode. @notsoocool

notsoocool commented:
> Okay can we like discuss it together, or do you have an idea in mind ? Like how it should look, so that its easier for me to understand

sure i was give you some examples

notsoocool commented:
<img width="902" alt="Screenshot 2024-10-22 at 12 42 06 AM" src="https://github.com/user-attachments/assets/a1390c10-5e6a-463c-8f56-a6c5898bd012">
<img width="929" alt="Screenshot 2024-10-22 at 12 42 23 AM" src="https://github.com/user-attachments/assets/d4897176-520d-4a31-87df-9eee4a5902b3">
<img width="986" alt="Screenshot 2024-10-22 at 12 42 43 AM" src="https://github.com/user-attachments/assets/e9b71ecb-4ecc-40e7-9ac4-994183d6d36d">

i find these types of design sleek and minimalist with small fonts and less text weight

notsoocool commented:
Hope you get what i am trying to say, if still you have some confusing we can further discuss it

dev129 commented:
A major problem the developement server is closing again and again , even upon refreshing or routing to pages. Please resolve this @notsoocool.
See this  

https://github.com/user-attachments/assets/a39507a5-66b1-4a0d-9ef8-8aa84dba440b

dev129 commented:
Hi @notsoocool , the header file div tag is fixed as I have stated about it , the issue here is that the developemetn server is closing again and agin by itself.

notsoocool commented:
> Hi @notsoocool , the header file div tag is fixed as I have stated about it , the issue here is that the developemetn server is closing again and agin by itself.

can you attach the ss of the starting of the error

notsoocool commented:
> > Hi @notsoocool , the header file div tag is fixed as I have stated about it , the issue here is that the developemetn server is closing again and agin by itself.

 can you attach the ss of the starting of the error
 because as i can see, when the faq is getting compiled the error comes after that

dev129 commented:
Well thanks alot @notsoocool , for making this bit easier as for a begineer like me its kinda diffcult to understand .

notsoocool commented:
@dev129 it's working perfectly fine for me. Do one thing: clone it once again, and now run it with **bun** instead of **npm**.

dev129 commented:
Hye @notsoocool , it worked miracously.

dev129 commented:
Hi @notsoocool 

https://github.com/user-attachments/assets/ea9d033b-4201-4bb0-b3d4-4b9a36190fe2

It is responsive to dark and light mode, design is made sleek and minmalist font size is reduced as some points any further reduction means a decline in User Experience, also added a feature so that if some cannot find the desired question then they can still ask for support, hence achieving optimisation.

notsoocool commented:
@dev129 this looks excellently good, I am impressed.

dev129 commented:
Great now what has to be done ?

notsoocool commented:
> Great now what has to be done ?

You haven't pushed the changes yet, all done for this issue now, just push the changes what you have made

dev129 commented:
okay

dev129 committed:
update FAQ Section

FAQ section is made sleek and minimalist and support for contact support is done to solve further question.

dev129 committed:
Upadte .env.sample

This is just an error

dev129 committed:
Merge branch 'develop' into feat-FAQ

dev129 commented:
Please check now @notsoocool , I hope now there are no further issues.

notsoocool committed:
revert some files back

notsoocool committed:
deleted unwanted image

notsoocool commented:
Thank you for your contribution! If you'd like to further support the project, feel free to star the repository—it helps the project reach more people and grow.
